src/benchmark_runner.cc

```diff
@@ -139,6 +139,7 @@ void RunInThread(const BenchmarkInstance* b, IterationCount iters,
     results.cpu_time_used += timer.cpu_time_used();
     results.real_time_used += timer.real_time_used();
     results.manual_time_used += timer.manual_time_used();
+    results.manual_time_used2 += timer.manual_time_used2();
```

Comment:
```suggestion
    results.manual_time_used_pow2 += timer.manual_time_used2();
```
I think it's a better name.

Reply:
(same remark for other `<...>2` that would become `<...>_pow2`)

Reply:
OK, done.